Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -105,14 +106,14 @@ public void ResetConfiguration(MixedRealityToolkitConfigurationProfile profile)
 
             activeProfile = profile;
 
-            if (profile != null)
-            {
-                if (Application.IsPlaying(profile))
-                {
-                    DisableAllServices();
-                }
-                DestroyAllServices();
-            }
+            //if (profile != null)
```

Comment:
Did you forget to remove this commented out code, or comment it out on purpose?

Reply:
forgot to remove. thanks!